compiler/IREmitter/IREmitterContext.cc

```diff
@@ -843,6 +872,11 @@ IREmitterContext IREmitterContext::getSorbetBlocks2LLVMBlockMapping(CompilerStat
                 userEntryBlockByFunction[ensureBlockId] = llvmBlocks[ensureBlock->id];
             }
         } else if (b->bexit.cond.variable.data(cfg)._name == core::Names::argPresent()) {
+            // TODO(trevor) we don't currently handle argPresent decisions in ruby-Argblocks, only for the method entry.
```

Comment:
I'm happy to remove this TODO, but my plan was to address it immediately in a follow-up PR.